mkt/ratings/tests/test_views.py

```diff
@@ -290,10 +289,10 @@ def test_filter_by_app_excluded_in_region(self, get_region_mock):
         AddonExcludedRegion.objects.create(addon=self.app,
                                            region=mkt.regions.BRA.id)
         get_region_mock.return_value = mkt.regions.BRA
```

Comment:
You can remove this test method `test_filter_by_app_excluded_in_region` entirely.